pcsx2/gui/Debugger/DisassemblyDialog.cpp

```diff
@@ -248,12 +248,12 @@ DisassemblyDialog::DisassemblyDialog(wxWindow* parent):
 	breakRunButton->Enable(false);
 	topRowSizer->Add(breakRunButton);
 
-	stepIntoButton = new wxButton( panel, wxID_ANY, L"Step Into");
+	stepIntoButton = new wxButton( panel, wxID_ANY, L"Step Into" + wxString(" (F11)"));
 	stepIntoButton->Enable(false);
 	Bind(wxEVT_BUTTON, &DisassemblyDialog::onStepIntoClicked, this, stepIntoButton->GetId());
 	topRowSizer->Add(stepIntoButton);
 
-	stepOverButton = new wxButton( panel, wxID_ANY, L"Step Over");
+	stepOverButton = new wxButton( panel, wxID_ANY, L"Step Over" + wxString(" (F10)"));
```

Comment:
Why + wxString? Can't you add it directly to the L"" string?
Also are the hotkeys configurable? If so you might want to parse the current key

Reply:
If i add it to the L string, it will affect translations and no they aren't configurable at this time, it's more a temporary solution.

Reply:
Ahhhhh ok, that surprisingly makes sense